pages/common/deemix.md

```diff
@@ -1,8 +1,7 @@
 # deemix
 
 > A barebone deezer downloader library built from the ashes of Deezloader Remix.
-> It can be used as a standalone CLI app or implemented in an UI using the API.
-> More information: <https://deemix.app>.
+> It can be used as a standalone CLI app or implemented in a UI using the API.
```

Comment:
```suggestion
> It can be used as a standalone CLI app or implemented in an UI using the API.
> More informattion: <https://deemix.app>
```
This one works again.

Reply:
Now we only make the `an` → `a` change here. Should we revert that too?

Reply:
[This one](https://github.com/tldr-pages/tldr/pull/5724#discussion_r657193280) also works again

Reply:
Looks good to me! Let's apply this change.